src/openforms/appointments/models.py

```diff
@@ -3,16 +3,12 @@
 
 from solo.models import SingletonModel
 
-from .constants import AppointmentDetailsStatus, AppointmentsConfigPaths
+from .constants import AppointmentDetailsStatus
+from .fields import AppointmentBackendChoiceField
 
 
 class AppointmentsConfig(SingletonModel):
-    config_path = models.CharField(
-        _("appointment plugin"),
-        choices=AppointmentsConfigPaths,
-        max_length=255,
-        blank=True,
-    )
+    config_path = AppointmentBackendChoiceField(_("appointment plugin"), blank=True)
```

Comment:
can we rename this field to `plugin` rather than `config_path`? I think that'll make the rest of the code a bit easier to grok